easybuild/easyblocks/n/neuron.py

```diff
@@ -60,85 +62,133 @@ def extra_options():
+            CMakeMake.configure_step(self)
 
     def install_step(self):
         """Custom install procedure for NEURON."""
 
         super(EB_NEURON, self).install_step()
 
-        if self.with_python:
-            pypath = os.path.join('src', 'nrnpython')
-            try:
-                pwd = os.getcwd()
-                os.chdir(pypath)
-            except OSError as err:
-                raise EasyBuildError("Failed to change to %s: %s", pypath, err)
+        # with the CMakeMake, the python module is installed automatically
+        if LooseVersion(self.version) < LooseVersion('7.8.1'):
+            if self.with_python:
+                pypath = os.path.join('src', 'nrnpython')
+                try:
+                    pwd = os.getcwd()
+                    os.chdir(pypath)
+                except OSError as err:
+                    raise EasyBuildError("Failed to change to %s: %s", pypath, err)
 
-            cmd = "python setup.py install --prefix=%s" % self.installdir
-            run_cmd(cmd, simple=True, log_all=True, log_ok=True)
+                cmd = "python setup.py install --prefix=%s" % self.installdir
+                run_cmd(cmd, simple=True, log_all=True, log_ok=True)
 
-            try:
-                os.chdir(pwd)
-            except OSError as err:
-                raise EasyBuildError("Failed to change back to %s: %s", pwd, err)
+                try:
+                    os.chdir(pwd)
+                except OSError as err:
+                    raise EasyBuildError("Failed to change back to %s: %s", pwd, err)
 
     def sanity_check_step(self):
         """Custom sanity check for NEURON."""
         shlib_ext = get_shared_lib_ext()
         binpath = os.path.join(self.hostcpu, 'bin')
         libpath = os.path.join(self.hostcpu, 'lib', 'lib%s.' + shlib_ext)
-        binaries = ["bbswork.sh", "hel2mos1.sh", "ivoc", "memacs", "mkthreadsafe", "modlunit", "mos2nrn",
-                    "mos2nrn2.sh", "neurondemo", "nocmodl", "oc"]
-        binaries += ["nrn%s" % x for x in ["gui", "iv", "iv_makefile", "ivmodl", "mech_makefile", "oc", "oc_makefile",
-                                           "ocmodl"]]
-        libs = ["ivoc", "ivos", "memacs", "meschach", "neuron_gnu", "nrniv", "nrnmpi", "nrnoc", "nrnpython", "oc",
-                "ocxt", "scopmath", "sparse13", "sundials"]
-
         # hoc_ed is not included in the sources of 7.4. However, it is included in the binary distribution.
         # Nevertheless, the binary has a date old enough (June 2014, instead of November 2015 like all the
         # others) to be considered a mistake in the distribution
+        binaries = []
         if LooseVersion(self.version) < LooseVersion('7.4'):
             binaries.append("hoc_ed")
 
-        custom_paths = {
+        extra_dirs = []
+        if LooseVersion(self.version) < LooseVersion('7.8.1'):
+            binaries = ["bbswork.sh", "hel2mos1.sh", "ivoc", "memacs", "mkthreadsafe", "modlunit", "mos2nrn",
```

Comment:
Did you mean to overwrite binaries here? I guess `+=` was intended, same below

Reply:
ah, indeed since I swapped the order of those two parts